src/python/pants/build_graph/build_configuration.py

```diff
@@ -180,6 +185,10 @@ def union_rules(self):
     """
     return self._union_rules
 
+  def target_adaptors(self):
```

Comment:
Nit: should this be a property? I'm not sure why the others like `rules()` aren't.